eras/babbage/test-suite/cardano-ledger-babbage-test.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-ledger-babbage-test
-version:       1.1.1.8
+version:       1.2
```

Nothing in this package chaneged, except cabal file, we can't bump any other version part than the patch
```suggestion
version:       1.1.1.9
```